cpp/src/plasma/protocol.cc

```diff
@@ -99,14 +103,15 @@ Status SendCreateReply(int sock, ObjectID object_id, PlasmaObject* object,
                                  object->device_num);
   auto object_string = fbb.CreateString(object_id.binary());
 #ifdef PLASMA_GPU
-  flatbuffers::Offset<CudaHandle> ipc_handle;
+  flatbuffers::Offset<flatbuf::CudaHandle> ipc_handle;
```

Comment:
When I was working with the IPC code, it was hard to distinguish between `flabuffers` and `flatbuf`. 
Later You might consider to use a namespace which implies that it belongs to arrow (something like `arrowfbs` or whatever instead of `flatbuf`)